diff --git a/runtime/filetype.vim b/runtime/filetype.vim
index a6f13c8..585e442 100644
--- a/runtime/filetype.vim
+++ b/runtime/filetype.vim
@@ -124,6 +124,9 @@ au BufNewFile,BufRead *.asy		setf asy
 au BufNewFile,BufRead .htaccess,*/etc/httpd/*.conf		setf apache
 au BufNewFile,BufRead */etc/apache2/sites-*/*.com		setf apache
 
+" Gentoo apache config file locations (Gentoo bug #83565)
+au BufNewFile,BufRead /etc/apache2/{modules,vhosts}.d/*.conf call s:StarSetf('apache')
+
 " XA65 MOS6510 cross assembler
 au BufNewFile,BufRead *.a65			setf a65
 
