diff --git a/runtime/syntax/grub.vim b/runtime/syntax/grub.vim
index 3743ae3..232a37c 100644
--- a/runtime/syntax/grub.vim
+++ b/runtime/syntax/grub.vim
@@ -27,7 +27,7 @@ syn match   grubBegin         display '^'
 
 syn cluster grubCommands      contains=grubCommand,grubTitleCommand
 
-syn keyword grubCommand       contained default fallback hiddenmenu timeout
+syn keyword grubCommand       contained default fallback hiddenmenu timeout splashimage
 
 syn keyword grubTitleCommand  contained title nextgroup=grubTitle skipwhite
 
